### DISCUSSION TITLE
How can remove the helps for typescript

### DISCUSSION THREAD

**Discussion by Agus9217** [LazyVim/LazyVim#3509] [Q&A] [answered]
Hi I update nvim and lazyvim and now I have some helps for typescript like the image, how I can remove it? thanks
<img width="330" alt="Captura de Pantalla 2024-06-06 a la(s) 21 28 52" src="https://github.com/LazyVim/LazyVim/assets/86172739/edf29226-17c5-4f92-8086-33bca03e08a3">

**Comment by dpetka2001** [comment, marked as answer]
In a file inside `/lua/plugins/` directory do the following
```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      inlay_hints = {
        enabled = false,
          exclude = {}, -- filetypes for which you don't want to enable inlay hints
        },
    },
  },
}
```
You can then use `<leader>uh` to toggle them on/off.

**Reply by Agus9217** [reply]
Thanks, it's runing good

**Reply by viktorvillalobos** [reply]
Thanks, I love you!
